docs/src/SUMMARY.md

```diff
@@ -1,17 +1,15 @@
 # Summary
 
 - [Introduction](./introduction.md)
-- [Contributing to Rust-GPU]()
+- Contributing to Rust-GPU
     - [Building](./building-rust-gpu.md)
     - [Testing](./testing.md)
     - ["Codegen args" (flags/options) supported by the Rust-GPU codegen backend](./codegen-args.md)
     - [Minimizing bugs in SPIR-V](./spirv-minimization.md)
     - [Publishing Rust-GPU on crates.io](./publishing-rust-gpu.md)
 - [Platform Support](./platform-support.md)
 - [Writing Shader Crates](./writing-shader-crates.md)
-- [Features]()
+- Features
     - [Attribute syntax](./attributes.md)
     - [Inline Assembly](./inline-asm.md)
     - [Image type syntax](./image.md)
-- [RFCs]()
-    - [001. Resource Binding Syntax](./rfcs/001-resource-binding-syntax.md)
```

Comment:
You removed this but not the underlying file, should we get rid of RFCs entirely? cc @repi

Reply:
I think it's fine having the RFC's in there indefinitely, but it felt wrong to keep such an old thing in the table of contents. It wouldn't really interest any user of rust-gpu.